### PR TITLE
Coredata tags

### PR DESCRIPTION
**Pull request by JB184351**

### CODE REVIEW

File: TalkAloud/TalkAloud/Model/AudioEngine.swift [hunk 1]
```diff
@@ -46,6 +46,7 @@ class AudioEngine: NSObject {
     
     // Intializing audioRecorder here to make clear
     // when I'm initializing the audioRecorder and actually recording
+    // TODO: Change fileURL to use attribute url
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/Model/AudioEngine.swift]: What do you mean with this TODO?

File: TalkAloud/TalkAloud/Model/AudioEngine.swift [hunk 2]
```diff
@@ -107,6 +108,7 @@ class AudioEngine: NSObject {
         audioState = .playing
     }
     
+    // TODO Change withFileURL to fetch url attribute
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/Model/AudioEngine.swift]: What do you mean with this TODO?

File: TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift [hunk 3]
```diff
@@ -78,7 +78,10 @@ class AudioPlayerViewController: UIViewController, AudioEngineStateChangeDelegat
     
     @IBAction func recordAudioButtonAction(_ sender: UIButton) {
         if AudioEngine.sharedInstance.audioState == .paused || AudioEngine.sharedInstance.audioState == .stopped {
-            AudioEngine.sharedInstance.setupRecorder(fileURL: AudioManager.sharedInstance.getNewRecordingURL())
+            // Use value forKey here to pass the url from the AudioRecording Object
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift]: What do you mean with this comment?

File: TalkAloud/TalkAloud/Model/AudioEngine.swift (hunk 1 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/Model/AudioEngine.swift]: I forgot I made this but I was thinking at first that I was going to have to use the url property from CoreData. I thought I deleted this maybe this was from that commit I went back to.

File: TalkAloud/TalkAloud/Model/AudioEngine.swift (hunk 2 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/Model/AudioEngine.swift]: I forgot I made this but I was thinking at first that I was going to have to use the url property from CoreData. I thought I deleted this maybe this was from that commit I went back to.

File: TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift (hunk 3 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift]: I forgot I made this but I was thinking at first that I was going to have to use the url property from CoreData. I thought I deleted this maybe this was from that commit I went back to.

**Review by virgil-adumitroae — APPROVED** [review]

**Pull request merged**